apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.html.tsx

```diff
@@ -144,230 +114,42 @@ export function AssessmentQuestion({
-                      disabled
-                    >
-                      <i class="fas fa-tags" /> Tag for grading
-                    </button>
-                    <div class="dropdown-menu dropdown-menu-end">
-                      <div class="dropdown-header">Assign for grading</div>
-                      {courseStaff.map((grader) => (
-                        <button
-                          key={grader.user_id}
-                          class="dropdown-item"
-                          type="submit"
-                          name="batch_action_data"
-                          value={JSON.stringify({
-                            requires_manual_grading: true,
-                            assigned_grader: grader.user_id,
-                          })}
-                        >
-                          <i class="fas fa-user-tag" />
-                          Assign to: {grader.name || ''} ({grader.uid})
-                        </button>
-                      ))}
-                      <button
-                        class="dropdown-item"
-                        type="submit"
-                        name="batch_action_data"
-                        value={JSON.stringify({ assigned_grader: null })}
-                      >
-                        <i class="fas fa-user-slash" />
-                        Remove grader assignment
-                      </button>
-                      <div class="dropdown-divider" />
-                      <button
-                        class="dropdown-item"
-                        type="submit"
-                        name="batch_action_data"
-                        value={JSON.stringify({ requires_manual_grading: true })}
-                      >
-                        <i class="fas fa-tag" />
-                        Tag as required grading
-                      </button>
-                      <button
-                        class="dropdown-item"
-                        type="submit"
-                        name="batch_action_data"
-                        value={JSON.stringify({ requires_manual_grading: false })}
-                      >
-                        <i class="fas fa-check-square" />
-                        Tag as graded
-                      </button>
-                    </div>
-                  </div>
-                )}
-              </div>
-            </div>
-            <table id="grading-table" aria-label="Instance questions for manual grading" />
-          </div>
-          <div
-            // eslint-disable-next-line @eslint-react/dom/no-dangerously-set-innerhtml
-            dangerouslySetInnerHTML={{
-              __html: GroupInfoModal({
-                modalFor: 'selected',
-                numOpenInstances: num_open_instances,
-                csrfToken: __csrf_token,
-              }).toString(),
-            }}
+        <Hydrate fullHeight>
+          <AssessmentQuestionManualGrading
+            authzData={authz_data}
```

Comment:
It looks like this and similar things come directly off `res.locals`. Surely we should be using the page-context machinery for this and similar things like `course` and `course_instance`?